semgrep/tests/conftest.py

```diff
@@ -74,6 +75,8 @@ def _run_semgrep(
     :param output_format: which format to use
     :param stderr: whether to merge stderr into the returned string
     """
+    possibly_notify_user()
```

Comment:
Having this called by the test code means that if we remove it from actual code path of semgrep tests will continue to believe the notification is being printed out no?

Reply:
@nbrahms

Reply:
We should add a comment; this just prevents the first-run output from showing up in the test output.

Also not really important for any of the tests that use this; it just allows me to avoid updating snapshots.